tempodb/backend/s3/s3.go

```diff
@@ -239,7 +239,9 @@ func (rw *readerWriter) Append(ctx context.Context, name string, keypath backend
 		a.partNum,
 		bytes.NewReader(buffer),
 		int64(len(buffer)),
-		minio.PutObjectPartOptions{},
+		minio.PutObjectPartOptions{
```

Comment:
Can we extract this as a new method in the same way we have the getPutObjectOptions?